{
  "candidate": "CVE-2020-14398",
  "description": "An issue was discovered in LibVNCServer before 0.9.13. An improperly closed TCP connection causes an infinite loop in libvncclient/sockets.c.",
  "priority": "medium",
  "cvss": {
    "nvd": {
      "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "score": 7.5,
      "severity": "HIGH"
    }
  },
  "references": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14398",
    "https://github.com/LibVNC/libvncserver/compare/LibVNCServer-0.9.12...LibVNCServer-0.9.13",
    "https://ubuntu.com/security/notices/USN-4434-1"
  ],
  "public_date": "2020-06-17T16:15:00Z",
  "public_date_at_usn": "2020-06-17T16:15:00Z",
  "packages": [
    {
      "name": "x11vnc",
      "status": "needs-triage"
    },
    {
      "name": "veyon",
      "status": "needs-triage"
    },
    {
      "name": "libvncserver",
      "status": "needs-triage",
      "patches": [
        {
          "source": "upstream",
          "text": "https://github.com/LibVNC/libvncserver/commit/57433015f856cc12753378254ce4f1c78f5d9c7b"
        }
      ]
    }
  ]
}
